MAUI/Segmented-Control/display-mode.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: post
+title: Display options in .NET MAUI Segmented control | Syncfusion
```

Comment:
title: Managing segment content  in .NET MAUI Segmented control
description: Learn about to manage the segment content in Syncfusion .NET MAUI Segmented(SfSegmented) control.

Reply:
changed title and description